src/Sinch/Conversation/Messages/Message/ListMessage.cs

```diff
@@ -95,6 +97,16 @@ public interface IListItem
     {
     }
 
+    public class ListItemChoiceWrapper : IListItem
```

Comment:
Do you think there is a way to hide the wrapper required onto JSON serialization only ?

The before/after samples from this PR (https://github.com/sinch/sinch-sdk-dotnet/pull/50/files#diff-e1497a7d723d2a4ac1227f7ef48afc83fe25163f5b2f07c76222feb1f7e2487fR48) highlight complexity of network protocol without added value for SDK's user.